README.md

```diff
@@ -195,6 +215,23 @@ Count number of events in each queue:
 jq 'map_values(map(keys[0]))' queue_dump.json
 ```
 
+### Debug console
+
+If the configuration option `console.enabled` is set to `true`, a unix socket named `debug.socket` by default can be found next to the configuration while the node is running. It can be connected to by tools like `socat`:
+
+```sh
+$ socat - unix:/path/to/debug.socket
+```
+
+Entering `help` will show available commands. The interface can also be scripted, it may be helpful to change some of the per-connection settings in this case:
+
+```sh
+echo -e 'set -q true -o bincode\nexample-command' | socat - unix-client:/path/to/debug.socket > output.dump
```

Comment:
Should it be 
```suggestion
echo -e 'set -q true -o bincode\example-command' | socat - unix-client:/path/to/debug.socket > output.dump
```
?

Reply:
No, the `\n` is a linebreak, which terminates the previous commands (thus the `-e` on `echo`).

Reply:
Ah, I thought it's a path 👍